src/ape/api/networks.py

```diff
@@ -22,28 +22,61 @@
 @abstractdataclass
 class EcosystemAPI:
     """
-    An Ecosystem is a set of related Networks
+    A set of related networks, such as Ethereum.
+    """
+
+    name: str
+    """
+    The name of the ecosystem. This should be set as the same name as the plugin.
     """
 
-    name: str  # Set as plugin name
     network_manager: "NetworkManager"
+    """A reference to the global network manager."""
+
     config_manager: "ConfigManager"
+    """A reference to  the global config manager."""
```

Comment:
nitpick: extra space 
```suggestion
    """A reference to the global config manager."""
```